lldb/source/Plugins/Process/wasm/ProcessWasm.cpp

```diff
@@ -0,0 +1,127 @@
+//===----------------------------------------------------------------------===//
```

Comment:
You're missing the filename in this line of all the license headers, unless this is a new format I'm not aware of.